frontend/src/stores/SongsStore.js

```diff
@@ -1,13 +1,11 @@
 import { Store } from 'flux/utils';
 import ApiService from '@Services/ApiService.js';
 import AppDispatcher from '@Services/AppDispatcher.js';
-import { updateSongsAction, updateSongAction, updateErrorAction, SongActions } from '@Actions/SongActions.js';
-import { List } from 'immutable';
+import { List, Record } from 'immutable';
 
-const Endpoints = {
```

Comment:
Deze Endpoints worden verderop in de code nog gebruikt :-) Denk dat je deze per ongeluk het weggegooid.

Reply:
Wss, nog ff kijken zo

Reply:
In dit script niet, die zit in een andere file

Reply:
In de __onDispatch() functie onder de GET_SONGS case onder andere :-) Als je nu naar de 'Nummers' pagina gaat, dan geeft de console een fout dat de Endpoints niet meer zijn gedefinieerd